src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -145,6 +145,8 @@ private void DisplayOutputResources(List<DisplayedResourceItem> displayedResourc
         /// <returns>A tuple consisting of the Orchestrator object, Selected Recommendation, Cloud Application metadata.</returns>
         public async Task<(Orchestrator, Recommendation, CloudApplication)> InitializeDeployment(string applicationName, UserDeploymentSettings? userDeploymentSettings, string deploymentProjectPath)
         {
+            string cloudApplicationName;

```

Comment:
nit - You don't need to declare this variable till you use the variable on line 176.